src/panels/lovelace/common/directives/action-handler-directive.ts

```diff
@@ -111,12 +111,10 @@ class ActionHandler extends HTMLElement implements ActionHandler {
         y = (ev as MouseEvent).pageY;
       }
 
-      if (options.hasHold) {
-        this.timer = window.setTimeout(() => {
-          this.startAnimation(x, y);
-          this.held = true;
-        }, this.holdTime);
-      }
+      this.timer = window.setTimeout(() => {
+        this.startAnimation(x, y);
+        this.held = true;
+      }, this.holdTime);
```

Comment:
I remember we discussed this somewhat; something to do with not wanting to show the ripple if there was no hold action set

Reply:
Yeah I still think this is not necessary when there is no hold action, but I also know @thomasloven had a lot of trouble getting this to work on all devices, so was not going to experiment 2 days before the release.

Reply:
absolutely